src/app/multi-line-chart/multi-line-chart.component.ts

```diff
@@ -185,6 +185,41 @@ export class MultiLineChartComponent implements OnInit {
           .attr('stroke-dashoffset', 0);
       });
 
+      // Grid lines
+      svg.append('g').call(g =>
+        g
+          .attr('stroke', 'black')
+          .attr('stroke-opacity', 0.1)
+          .call(g =>
+            g
+              .append('g')
+              .selectAll('line')
+              .data(
+                [10, 20, 30, 40, 50, 60, 70, 80, 90, 100]
```

Comment:
ideally this would be based on the max length of any location's series, right?

Reply:
yep!